Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -137,10 +137,10 @@ is ready to operate:
     cilium-mkcmb        1/1       Running   0          10m
 
 Note, in above Helm configuration, the ``kubeProxyReplacement`` has been set to
-``strict`` mode. This means that the Cilium agent will bail out in case the
+``true`` mode. This means that the Cilium agent will bail out in case the
 underlying Linux kernel support is missing.
 
-By default, Helm sets ``kubeProxyReplacement=disabled``, which only enables
+By default, Helm sets ``kubeProxyReplacement=false``, which only enables
```

Comment:
nit, I'd add: In order to support older kernels than v4.19, by default, ...

Reply:
Wouldn't this be a bit misleading? E.g., there are some rough corners with Socket LB.